### PR TITLE
Deprecate oc types

### PR DESCRIPTION
**Pull request by soltysh**
/assign @mfojtik

### CODE REVIEW

File: pkg/oc/cli/types/types.go [hunk 1]
```diff
@@ -227,11 +227,12 @@ func NewCmdTypes(fullName string, f kcmdutil.Factory, streams genericclioptions.
...
-		Use:     "types",
-		Short:   "An introduction to concepts and types",
-		Long:    fmt.Sprintf(typesLong, buf.String()),
-		Example: fmt.Sprintf(typesExample, fullName),
-		Run:     kcmdutil.DefaultSubCommandRun(streams.Out),
+		Use:        "types",
+		Short:      "An introduction to concepts and types",
+		Long:       fmt.Sprintf(typesLong, buf.String()),
+		Example:    fmt.Sprintf(typesExample, fullName),
+		Deprecated: "use official docs instead",
```
Comment (mfojtik) [inline pkg/oc/cli/types/types.go]: "please refer to official documentation instead"

**Comment by mfojtik** [comment]
/lgtm

Need release note update.

**Comment by soltysh** [comment]
> Need release note update.

Already there.

**Pull request merged**